docs/API.md

```diff
@@ -1920,6 +1920,98 @@ if err != nil {
 }
 ```
 
+<a name="SetBucketEncryption"></a>
+### SetBucketEncryption(bucketname, configuration string) error
+Set default encryption configuration on a bucket.
+
+__Parameters__
+
+|Param   |Type   |Description   |
+|:---|:---| :---|
+|`bucketName` | _string_  |Name of the bucket|
+|`configuration` | _string_  |Default encryption configuration to be set |
+
+__Return Values__
+
+|Param   |Type   |Description   |
+|:---|:---| :---|
+|`err` | _error_  |Standard Error   |
+
+__Example__
+
+```go
+encryptionConfig := `<ServerSideEncryptionConfiguration
+    xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
+    <Rule>
+        <ApplyServerSideEncryptionByDefault>
+            <SSEAlgorithm>AES256</SSEAlgorithm>
+        </ApplyServerSideEncryptionByDefault>
+    </Rule>
+</ServerSideEncryptionConfiguration>`
+
+err = minioClient.SetBucketEncryption("my-bucketname", encryptionConfig)
```

Comment:
This example needs to be updated @ebozduman

Reply:
ok